public/js/graph/react_graph.js

```diff
@@ -934,24 +940,26 @@ var EdgeGroup = React.createClass({
     render: function () {
     	var missingEdges = [];
     	var otherEdges = [];
-    	var missEdgesDict = this.state.missEdges;
     	var edges = this.props.edgesJSON;
-    	// console.log(Object.prototype.toString.call(edges[0]));
-		// console.log('missing edges dict', missEdgesDict);
 		for (var i = 0; i < edges.length; i++) {
 			// if (this.refs[edges[i]] !== undefined) {
 			// console.log(edges[i].id_);
-			if (edges[i].id_ in missEdgesDict) {
-				var edgeID = edges[i].id_;
-	      		if (missEdgesDict[edgeID] !== true) {
-	      			otherEdges.push(edges[i]);
-	      		} else {
-	     			missingEdges.push(edges[i]);
-	     		}
+			var edgeID = edges[i].id_;
+			// console.log(this.state);
+			if (edgeID in this.state) {
```

Comment:
I don't think you ever remove edges from the state. So you should be checking not only for whether the state has the edgeID, but whether `this.state[edgeID]` is missing (or something like that.